python/cudf_polars/cudf_polars/testing/asserts.py

```diff
@@ -81,7 +89,7 @@ def assert_gpu_result_equal(
     )
 
     expect = lazydf.collect(**final_polars_collect_kwargs)
-    engine = GPUEngine(raise_on_fail=True)
+    engine = GPUEngine(raise_on_fail=True, executor=Executor)
```

Comment:
Should this be something like `executor=executor or Executor`?
Right now, it seems like the `executor` is always ignored.